src/Extract_Data.py

```diff
@@ -132,6 +132,7 @@ def rescale_g_r(extracted_g_r_data):
 def locate_peaks(g_r):
     """
     Locate the peaks (maxima) in G(r) data within a specified range.
+    #Maybe an extra sentence here to inform where function find_peaks is defined? Is this a built-in function?
```

Comment:
Good point.  Yes, the peaks are determined via the built-in function find_peaks, imported from scipy.signal.